### PR TITLE
Use debian target for audio

### PR DESCRIPTION
**Pull request by Timple**
Rokus' commit is accepted: https://github.com/ros-drivers/audio_common/pull/93

Debian build is from 0.3.1-0xenial-20170613-153511-0800

So seems merged but needs to be confirmed.

Lets wait until after RoboCup

### CODE REVIEW

File: installer/targets/ros-audio_capture/install.yaml [hunk 1]
```diff
@@ -1,6 +1,5 @@
 - type: ros
   source:
     type: git
```
Comment (MatthijsBurgh) [inline installer/targets/ros-audio_capture/install.yaml]: this line needs to removed

Comment (Timple) [reply installer/targets/ros-audio_capture/install.yaml]: True!

**Comment by MatthijsBurgh** [comment]
@Timple let's merge?

File: installer/targets/ros-audio_capture/install.bash [hunk 2]
```diff
@@ -1,18 +1,18 @@
...
+#for dep in $deps
+#do
+#    if [ $dep != "Depends:" ] && [ $dep != "ros-$ROS_DISTRO-audio-capture" ] && [ $dep != "ros-$ROS_DISTRO-audio-common-msgs" ] && [ $dep != "ros-$ROS_DISTRO-roscpp" ]
+#    then
+#        #echo $dep
+#        targets="$targets $dep"
+#    fi
+#done
+##echo $targets
+#tue-install-system $targets
```
Comment (Rayman) [inline installer/targets/ros-audio_capture/install.bash]: Just remove the file if everything is commented.

Comment (MatthijsBurgh) [reply installer/targets/ros-audio_capture/install.bash]: I wanted it to keep in case the debian pkg is not correct, but that is fixed by a revert.

**Comment by Rayman** [comment]
If you rebase the last 3 commits into 1 I'll accept this (I can show you tonight how to do this). Has this been tested?

**Pull request merged**